src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Controllers/PolicyController.cs

```diff
@@ -10,15 +15,76 @@ namespace Altinn.Platform.Authorization.Controllers
     [ApiController]
     public class PolicyController : ControllerBase
     {
+        private readonly ILogger<PolicyController> logger;
         private readonly IPolicyRetrievalPoint _prp;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="PolicyController"/> class.      
         /// </summary>
         /// <param name="prp">The policy retrieval point.</param>
-        public PolicyController(IPolicyRetrievalPoint prp)
+        /// <param name="logger">logger</param>
+        public PolicyController(
+            IPolicyRetrievalPoint prp,
+            ILogger<PolicyController> logger)
         {
             _prp = prp;
+            this.logger = logger;
+        }
+
+        /// <summary>
+        /// Saves policy file to storage    
+        /// </summary>
+        /// <param name="org">Unique identifier of the organisation responsible for the app.</param>
+        /// <param name="app">Application identifier which is unique within an organisation.</param>
+        [HttpPost]
+        public async Task<ActionResult> WritePolicy([FromQuery] string org, [FromQuery] string app)
+        {
+            if (string.IsNullOrWhiteSpace(org))
+            {
+                return BadRequest("Organisation must be defined in query string");
+            }
+
+            if (string.IsNullOrWhiteSpace(app))
+            {
+                return BadRequest("App must be defined in query string");
+            }
+
```

Comment:
Should we also add verification of the format so we don't break the solution trying to parse the wrong format as xml when retrieving the policy ?

Reply:
#2912  created technical debt issue